index.html

```diff
@@ -145,44 +145,60 @@ <h4 data-ja="一次情報源">Primary Data Sources</h4>
     if(typeof drawMap === 'undefined'){
       return
     }
-    
+
     initDataTranslate()
     drawMap()
-    
+
+    var draws = 0;
     var styleLoaded = false
     var jsonData = undefined
-    
+    var FIVE_MINUTES_IN_MS = 300000
```

Comment:
Nitpick: should this be a `const`?

Reply:
Yes, I didn't know that level of IE support was wanted, so I was used the existing convention. But, updated to `const`

Reply:
Good call. Looks like at least IE11 supports it (not IE10). Future babel support coming anyway 👌